engine/pkg/meta/internal/sqlkv/model/model.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlkv
+
+import (
+	metaModel "github.com/pingcap/tiflow/engine/pkg/meta/model"
+	ormModel "github.com/pingcap/tiflow/engine/pkg/orm/model"
+)
+
+const (
+	// MetaKVTableName is the explicit defined table name for metakv
+	// We do this to avoid wrong interpretation of table name when jobID is empty
```

Comment:
The sentence can be improved:
- What does "this" refer to in "We do this to..."?
- wrong interpretation -> misinterpretation